packages/builders/__tests__/messages/poll.test.ts

```diff
@@ -7,22 +7,33 @@ const dummyData = {
 		text: '.',
 	},
 	answers: [],
-};
+} satisfies RESTAPIPoll;
+
+const dummyDataWithAnswer = {
+	...dummyData,
+	answers: [
+		{
+			poll_media: {
+				text: '.',
+			},
+		},
+	],
+} satisfies RESTAPIPoll;
 
 describe('Poll', () => {
 	describe('Poll question', () => {
 		test('GIVEN a poll with pre-defined question text THEN return valid toJSON data', () => {
-			const poll = new PollBuilder({ question: { text: 'foo' } });
```

Comment:
Shouldn't this be used as a test that would reject now because of missing answers? To test the new assertion